docs/user/whats-new.asciidoc

```diff
@@ -1,85 +1,63 @@
 
 [discrete]
 === Discover and ES|QL
 
 [discrete]
-==== Contextual Data presentation
+==== Improving column reordering with draggable columns
 
-In this release, Discover introduces enhanced contextual data presentation. Previously, you needed to manually select relevant fields and set up your workspace before diving into data exploration. Now, Discover automatically tailors the user experience based on the data being explored, powered by a scalable contextual architecture. For example, when analyzing logs, you'll see a *log.level* field rendered directly in the table, a custom Logs overview in the document viewer, and log.level indicators on individual rows. 
+We’re excited to announce an enhancement to the Discover table. You can now reorder columns by simply dragging them directly in the data table header. This intuitive feature reduces the number of interactions required to adjust your column layout by eliminating the need to open the Columns popover for sorting. This enhancement saves users valuable time and improves overall usability, leading to smoother and more efficient data exploration. 
 
-image::images/discover-log-level.png[Log level badge displaying in the Discover grid]
+image::https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt6fa3a9ee893e5603/67533de0e995a041b8dea1d9/dragging-columns-in-discover.gif[Dragging a column to adjust column layout in Discover]
 
 [discrete]
-==== Recommended ES|QL queries
-
-Writing ES|QL queries just got easier. Many users face challenges when authoring queries, and even more so when unfamiliar with the syntax or data structure. This can lead to inefficiencies in data analysis and visualization. We want to reduce the time it takes to create queries and to lower the learning curve for both new and existing users by suggesting recommended queries within the ES|QL Help menu and from the auto-complete.
-
-image::images/esql-suggestions.png[A list of suggestions to get started with an ES|QL query, width=30%]
-_Recommended ES|QL queries from the ES|QL help menu_
-
-image::images/esql-autocomplete-suggestions.png[A list of suggestions in the autocomplete menu of an ES|QL query, width=50%]
-_Recommended ES|QL queries from auto-complete suggestions_
+==== Starred ES|QL queries for quick access and reusability
 
+With this release, you can favorite your ES|QL queries directly from your query history by starring them. These starred queries are saved under a dedicated Starred tab, allowing you quick access to your most-used queries across Kibana – not just within Discover. With the ability to manage up to 100 starred queries per user, you can sort them by timestamp and receive warnings as you approach the limit. This enhancement utilizes the favorites service at the user level, enabling users to reuse ES|QL queries in other areas of Kibana, such as the dashboard in-line editor and alerts.
```

Comment:
> With this release, you can favorite your ES|QL queries directly from your query history by starring them. 

Is there a more common verb we ought to use here instead of "favorite"? e.g. bookmark?

Reply:
Good point. I've changed it to "bookmark"